src/lib/courses.ts

```diff
@@ -34,9 +34,9 @@ const courseQuery = groq`
     'avatar_url': person->image.url,
     'slug': person->slug.current
   },
-	'dependencies': softwareLibraries[]{
+	'dependencies': topicTaggings[]{
```

Comment:
Any particular reason for `topicTaggings` instead of `topicTags`?

Reply:
a "tagging" would be a join table and not the tag itself so it would represent a relationship

I don't know what's appropriate here